static/js/timerender.ts

```diff
@@ -24,6 +24,20 @@ export function clear_for_testing(): void {
     next_timerender_id = 0;
 }
 
+// by providing a default string of 'PP' or any of its variants for `formatStr`
+// it will format dates in whichever way is appropriate to the locale
+function localizedFormat(date: Date | number, formatOptions?: Intl.DateTimeFormatOptions): string {
```

Comment:
I think since this file handles both dates and times, and this is just for dates, we should name it `localizedDateFormat` or something to that effect?